### PR TITLE
Update UML diagrams

### PR DESCRIPTION
**Pull request by Pluiexo**
Part of #103 
The follow diagrams and images has been updated:
- StorageClassDiagram
- ModelClassDiagram

Changed traces of addressbook as well.

### CODE REVIEW

File: docs/images/ModelClassDiagram.png
Comment (tsulim) [inline docs/images/ModelClassDiagram.png:1]: The picture doesn't match with the content of ModelClassDiagram.puml. `* filtered` and `* all` are shown flipped.

**Review by tsulim — COMMENTED** [review]
Overall, it looks good to me. However, I noticed that one of the diagram image doesn't correspond with its puml content.

File: docs/images/ModelClassDiagram.png
Comment (Pluiexo) [reply docs/images/ModelClassDiagram.png:1]: I noticed that earlier, but didn't know whats wrong with the plugin. So I had to switch to windows to generate the image. My guess is probably just the graphviz library difference in mac

**Review by tsulim — APPROVED** [review]
LGTM! 👍

**Pull request merged**